Sources/LanguageServerProtocol/SupportTypes/TestItem.swift

```diff
@@ -56,12 +56,16 @@ public struct TestItem: ResponseType, Equatable {
   /// Tags associated with this test item.
   public var tags: [TestTag]
 
+  /// Whether the `TestItem` is declared in an extension.
+  public var isExtension: Bool
```

Comment:
I’d prefer to not add this to `LanguageServerProtocol.TestItem` because I don’t want to expose it as part of the LSP response, which we should keep stable. It probably means that we need to duplicate this struct for internal processing but … that’s what it is.